src/basicOp.js

```diff
@@ -24,7 +24,18 @@
  * @return {number}
  */
 function basicOp(operation, value1, value2) {
-  // write code here
+  switch (operation) {
+    case '+' :
+      return value1 + value2;
+    case '-' :
+      return value1 - value2;
+    case '*' :
+      return value1 * value2;
+    case '/' :
+      return value1 / value2;
+  }
+
+  return 0;
```

Comment:
My bad. You should use `default` in the switch case.

Reply:
i forgot about this feature)